spec/support/callback_modifier.rb

```diff
@@ -0,0 +1,28 @@
+class BeforeDestroyModifier < CallbackModifier
+  has_paper_trail
+  paper_trail_on_destroy :before
+end
+
+class AfterDestroyModifier < CallbackModifier
+  has_paper_trail
+  paper_trail_on_destroy :after
+end
+
+class NoArgDestroyModifier < CallbackModifier
+  has_paper_trail
+  paper_trail_on_destroy
+end
+
+class UpdateModifier < CallbackModifier
+  paper_trail_on_update
+end
+
+class CreateModifier < CallbackModifier
+  paper_trail_on_create
+end
+
+class DefaultModifier < CallbackModifier
+  # Because of the way I set up the destroy method for testing
+  # has_paper_trail has to be initialized in this model seperately
+  has_paper_trail
+end
```

Comment:
Because for me they are more like helper-classes wrapped around the model, it didn't feel right to put them there with all the "normal" activerecord models.